<launch>
  <!-- R2, SE2, R3, or SE3 are supported -->
  <arg name="sampling_space" default="R2" />
  <arg name="load_grid" default="false" />

  <node pkg="differentiable_rmap" type="NodeRmapVisualization" name="rmap_visualization"
        output="screen">
    <rosparam subst_value="true">
      sampling_space: $(arg sampling_space)
      config_path: $(find differentiable_rmap)/config/RmapVisualization.yaml
      sample_bag_path: /tmp/rmap_sample_set_$(arg sampling_space).bag
      svm_path: /tmp/rmap_svm_model_$(arg sampling_space).libsvm
      grid_bag_path: /tmp/rmap_grid_set_$(arg sampling_space).bag
      load_grid: $(arg load_grid)
    </rosparam>
    <remap from="marker_arr" to="visualization/marker_arr" />
  </node>

  <node pkg="rviz" type="rviz" name="$(anon rviz)"
        args="-d $(find differentiable_rmap)/rviz/RmapVisualization.rviz"
        output="log" clear_params="true" />
</launch>
